include/seastar/core/metrics_api.hh

```diff
@@ -391,7 +391,7 @@ struct options : public program_options::option_group {
 /*!
  * \brief set the metrics configuration
  */
-future<> configure(const options& opts);
+future<> configure(const options& opts, int handle = impl::default_handle());
```

Comment:
nit: Perhaps there should be a:
```c++
namespace seastar::metrics {

int seastar::metrics::default_handle() {
    return impl::default_handle();
}

}}
```
For the public api?

Reply:
There are still a few calls to `impl::default_handle()`

Reply:
Thanks for pointing it out. Should be fine now.